roles/openshift_examples/files/examples/x86_64/image-streams/dotnet_imagestreams.json

```diff
@@ -14,134 +14,197 @@
+                            "tags": "builder,.net,dotnet,dotnetcore,hidden",
                             "supports": "dotnet",
-                            "sampleRepo": "https://github.com/redhat-developer/s2i-dotnetcore-ex.git",
+                            "sampleRepo": "https://github.com/redhat-developer/s2i-dotnetcore-ex",
                             "sampleContextDir": "app",
-                            "sampleRef": "dotnetcore-2.2"
+                            "sampleRef": "dotnet-5.0"
                         },
                         "referencePolicy": {
                             "type": "Local"
                         },
                         "from": {
                             "kind": "ImageStreamTag",
-                            "name": "2.2"
+                            "name": "5.0-ubi8"
                         }
                     },
                     {
-                        "name": "2.2",
+                        "name": "5.0-ubi8",
                         "annotations": {
-                            "openshift.io/display-name": ".NET Core 2.2",
-                            "description": "Build and run .NET Core 2.2 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/redhat-developer/s2i-dotnetcore/tree/master/2.2/build/README.md.",
+                            "openshift.io/display-name": ".NET 5 (UBI 8)",
+                            "description": "Build and run .NET 5 applications on UBI 8. For more information about using this builder image, including OpenShift considerations, see https://github.com/redhat-developer/s2i-dotnetcore/tree/master/5.0/build/README.md.",
                             "iconClass": "icon-dotnet",
-                            "tags": "builder,.net,dotnet,dotnetcore,rh-dotnet22",
-                            "supports": "dotnet:2.2,dotnet",
-                            "sampleRepo": "https://github.com/redhat-developer/s2i-dotnetcore-ex.git",
+                            "tags": "builder,.net,dotnet,dotnetcore,dotnet50",
+                            "supports": "dotnet:5.0,dotnet",
+                            "sampleRepo": "https://github.com/redhat-developer/s2i-dotnetcore-ex",
                             "sampleContextDir": "app",
-                            "sampleRef": "dotnetcore-2.2",
-                            "version": "2.2"
+                            "sampleRef": "dotnet-5.0",
+                            "version": "5.0"
                         },
                         "referencePolicy": {
                             "type": "Local"
                         },
                         "from": {
                             "kind": "DockerImage",
-                            "name": "registry.redhat.io/dotnet/dotnet-22-rhel7:2.2"
+                            "name": "registry.access.redhat.com/ubi8/dotnet-50:5.0"
```

Comment:
@spolti Can you comment as to why some of these image names were switch to the old host (registry.access.redhat.com)?  This has caused a bug and these image streams should probably be corrected for the new host.  https://bugzilla.redhat.com/show_bug.cgi?id=1964084

Reply:
THanks for the heads up.